include/amici/solver.h

```diff
@@ -1163,11 +1372,11 @@ class Solver {
     StateOrdering ordering = StateOrdering::AMD;
 
     /** maximum number of allowed Newton steps for steady state computation */
-    int newton_maxsteps = 0;
+    long int newton_maxsteps = 0;
```

Comment:
Is this explicitly long int in sundials, or something like sun_index_type? The latter would be more flexible.

Reply:
explicitely `long int`